scalapact-shared/src/main/scala/com/itv/scalapact/shared/PendingPactSettings.scala

```diff
@@ -0,0 +1,27 @@
+package com.itv.scalapact.shared
+
+import java.time.OffsetDateTime
+
+sealed abstract case class PendingPactSettings(
+    enablePending: Option[Boolean],
+    includeWipPactsSince: Option[OffsetDateTime]
+) {
+  def append(that: PendingPactSettings): PendingPactSettings = that match {
```

Comment:
Sort of related to the other comment but this append behaviour feels a bit confusing as it's preferring `that` over `this` and is sort of duplicating some of the parsing logic from elsewhere. Guess this is because It diverges from the pattern of having an `Option[X]` for the settings in `ScalaPactSettings` - could following that pattern simplify it? Maybe with the ADT idea